demosplan/DemosPlanCoreBundle/Exception/DuplicatedTagTitleException.php

```diff
@@ -10,15 +10,41 @@
 
 namespace demosplan\DemosPlanCoreBundle\Exception;
 
+use demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure;
 use Exception;
+use demosplan\DemosPlanCoreBundle\Entity\Statement\TagTopic;
 
 class DuplicatedTagTitleException extends Exception
 {
+    public function __construct(string                       $message,
+                                protected readonly Procedure $procedure,
```

Comment:
`Procedure` is not needed

Reply:
i ll fix it, i think i did another mistake here which i have to check, this way the message bag will be showed only while importing, the goal was to use it for the other case too ( while manual tag creating ).

Reply:
fixed in https://github.com/demos-europe/demosplan-core/pull/2018/commits/6a030eb8aa1a3db49fa4be05a24639d2972c2fd9